### PR TITLE
Add a way to view raw content of markdown files

### PR DESCRIPTION
**Pull request by Tunous**
A small feature that I needed. Allows you to see a raw content of any markdown file:

peek 2017-08-08 19-40

### CODE REVIEW

File: app/src/main/java/com/gh4a/activities/FileViewerActivity.java [hunk 1]
```diff
@@ -258,6 +266,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 startActivity(CommitHistoryActivity.makeIntent(this,
                         mRepoOwner, mRepoName, mRef, mPath));
                 return true;
+            case R.id.view_raw:
+                boolean newState = !mViewRawText;
+                item.setChecked(newState);
+                mViewRawText = newState;
```
Comment (maniac103) [inline app/src/main/java/com/gh4a/activities/FileViewerActivity.java]:
```
mViewRawText = !mViewRawText;
item.setChecked(mViewRawText);
```

?

File: app/src/main/java/com/gh4a/activities/FileViewerActivity.java [hunk 2]
```diff
@@ -295,6 +309,11 @@ protected void onLineTouched(int line, int x, int y) {
         popupMenu.setOnMenuItemClickListener(this);
     }
 
+    @Override
+    protected boolean shouldWrapLines() {
+        return !(FileUtils.isMarkdown(mPath) && !mViewRawText) && super.shouldWrapLines();
+    }
```
Comment (maniac103) [inline app/src/main/java/com/gh4a/activities/FileViewerActivity.java]:
```
boolean displayingMarkdown = FileUtils.isMarkdown(mPath) && !mViewRawText;
return !displayingMarkdown && super.shouldWrapLines();
```

**Comment by maniac103** [comment]
Feature totally makes sense to me, just got some minor code simplification suggestions - see comments.

**Comment by Tunous** [comment]
Updated.

**Pull request merged**